pwa/src/styling/design-tokens/component-overrides.css

```diff
@@ -6,9 +6,6 @@
     --web-app-secondary-top-nav-font-size
   );
   --conduction-secondary-top-nav-color: var(--web-app-secondary-top-nav-color);
-  --conduction-secondary-top-nav-background-color-hover: var(
-    --web-app-secondary-top-nav-background-color-hover
-  );
```

Comment:
See the explanation comment about `.prettier` in your other pull request, the same issue is happening here.